config/environments/production.rb

```diff
@@ -22,7 +22,7 @@
       port:           '587',
       address:        'smtp.mandrillapp.com',
       user_name:      ENV['MANDRILL_USERNAME'],
-      password:       ENV['MANDRILL_API_KEY'],
+      password:       ENV['MANDRILL_APIKEY'],
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.